### PR TITLE
Add tbt reader for the SPS

### LINKED ISSUE #128: Add SPS-TBTReader
Should be similar to: https://github.com/pylhc/Beta-Beat.src/blob/df93c2691193f1258e78d240508a3354517bff11/MODEL/SPS/Tools/convert_0.2.py

We don't have an example file, so maybe look in /nfs/cs-ccr-nfs4/sps_data/OP_DATA/multit/betabeat/2012-11-08/SPS/Measurements

As no one has used this since 2017, this issue has HIGH PRIORITY!!!

### PR DESCRIPTION
**Pull request by Mael-Le-Garrec**
Fixes #128

### CODE REVIEW

**Comment by Mael-Le-Garrec** [comment]
SPS SDDS files are not completely agreed upon yet, hence why this PR is a draft for now

File: omc3/tbt/reader_sps.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    nbpms = len(bpm_names)
+    nchannels = len(channel_names)
+
+    data = {'X': [], 'Y': []}
+    inverted_plane = {'X': 'Y', 'Y': 'X'}
+    for channel in channel_names:
+        plane = PLANE_CORRES[channel[-1]]
+
+        data[plane].extend(sdds_file.values[channel])
+        data[inverted_plane[plane]].extend([0] * len(sdds_file.values[channel]))
```
Comment (mihofer) [inline omc3/tbt/reader_sps.py:60]: looking into the [model](https://gitlab.cern.ch/acc-models/acc-models-sps/-/blob/2021/SPS_LS2_2020-05-26.seq) and the sdds, SPS apparently has only single plane BPMs, when reading in the other plane is then filled with zeros ..
could become problematic when cleaning with keep_exact_zeros=False and further processing ..

**Comment by fsoubelet** [comment]
> SPS SDDS files are not completely agreed upon yet, hence why this PR is a draft for now

Any update on this @Mael-Le-Garrec?

**Comment by Mael-Le-Garrec** [comment]
The `tbt` code moved to the package `turn_by_turn`.
The SPS reader was implemented by this PR https://github.com/pylhc/turn_by_turn/pull/9

**Comment by JoschD** [comment]
whoops. I didn't even know that this was already here. Sorry!

**Pull request closed without merging**